src/wp-includes/default-filters.php

```diff
@@ -603,6 +603,14 @@
 // Global styles custom CSS.
 add_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles_custom_css' );
 
+// Block template.
+add_filter( 'the_block_template_html', 'shortcode_unautop', 7 );
+add_filter( 'the_block_template_html', 'do_shortcode', 8 );
```

Comment:
While this maintains the original order of the function, I wonder whether we could align this with the filter order execution on the other similar scenarios, e.g. on `the_content`. On those, `shortcode_unautop` uses the default `10` and `do_shortcode` uses `11`.

I know this comment is a bit opposite of my other comment, but I'm purely throwing it out there for consideration. I wonder why it was even decided in the first place to use a different order here.

On another note: If we used `10` and `11` here, it would also allow to use `8` for the filters in the `WP_Embed` class, which is another place to make things more consistent.
```suggestion
add_filter( 'the_block_template_html', 'shortcode_unautop' );
add_filter( 'the_block_template_html', 'do_shortcode', 11 );
```